CONTRIBUTING.md

```diff
@@ -0,0 +1,404 @@
+* [f-droid](https://f-droid.org/en/packages/com.nextcloud.talk2/)
+
+
+## Types
+We do differentiate between three different kinds of releases:
+
+
+### Stable
+Play store and f-droid releases for the masses.
+Pull Requests that have been tested and reviewed can go to master. After the last alpha release is out in the wild and no mayor errors get reported (by users or in the developer console) the master branch is ready for the stable release phase.
+So when we decide to go for a new release we freeze the master feature wise and a stable branch will be created.
+
+### Release Candidate
+_stable beta_ releases done via the Beta program of the Google Play store.
+Whenever a PR is reviewed/approved we put it on master.
+Before releasing a new stable version there is at least one release candidate. It is based on the current stable-branch. After a beta testing phase a stable version will be released, which is identical to the latest release candidate.
+
+
+### Alpha Release
+_alpha_ releases done via the Alpha program of the Google Play store.
+Whenever a PR is reviewed/approved we put it on master.
+Alpha releases are based on latest master and and we aim to release a new alpha version on a weekly basis. 
+
+
+### QA Release
+Done as a standalone app that can be installed in parallel to the stable app.
+Any PR gets a QA build so users and reporters are able to easily test a change (feature or bugfix.
+
+
+## Version Name and number
+### Stable / Release candidate
+For _stable_ and _release candidate_ the version name follows the [semantic versioning schema](http://semver.org/) and the version number has several digits reserved to parts of the versioning schema inspired by the [jayway version numbering](https://www.jayway.com/2015/03/11/automatic-versioncode-generation-in-android-gradle/), where:
+
+* 2 digits for beta/alpha code as in release candidates starting at '01' (1-50=Alpha / 51-89=RC / 90-99=stable)
+* 2 digits for hot fix code
+* 3 digits for minor version code
+* n digits for mayor version code
+
+![Version code schema](/docs/semantic_versioning_code.png "Semantic versioning code")
+
+Examples for different versions:
+* 1.0.0 ```10000099```
+* 8.12.2 ```80120290```
+* 9.8.4-Alpha18 ```90080418```
+* 11.2.0-rc1 ```110020051```
+
+beware, that beta releases for an upcoming version will always use the minor and hotfix version of the release they are targeting. So to make sure the version code of the upcoming stable release will always be higher stable releases set the 2 beta digits to '90'-'99' as seen above in the examples. For major versions, as we're not a library and thus 'incompatible API changes' is not something that happens, decisions are essentially marketing-based. If we deem a release to be very impactful, we might increase the major version number.
+
+
+## Release cycle
+* Releases are planned every ~2 months, with 6 weeks of developing and 2 weeks of stabilising
```

Comment:
for major releases it's more like ~4 months because we have the same release cycle like the server app
https://apps.nextcloud.com/apps/spreed/releases?platform=21#21

Reply:
I'll add another line for the link between server/client releases :)